.drone.star

```diff
@@ -2,23 +2,23 @@ def main(ctx):
   versions = [
     {
       'value': 'latest',
-      'qa': 'https://download.owncloud.com/internal/10.3.2/testing/owncloud-enterprise-complete-20191205-qa.tar.bz2',
-      'tarball': 'https://download.owncloud.com/internal/10.3.2/owncloud-enterprise-complete-20191205.tar.bz2',
-      'tarball_sha': 'b3d880b3d1aec6834527fa3e671cf931c9d3b42150f68071ed29021e724b54f7',
+      'qa': 'https://download.owncloud.com/internal/10.4.0/owncloud-enterprise-complete-10.4.0-qa.tar.bz2',
+      'tarball': 'https://download.owncloud.com/internal/10.4.0/owncloud-enterprise-complete-10.4.0.tar.bz2',
+      'tarball_sha': 'a6256666dfda0e2543e095bef94b3ca1b4b8203faf1a65b54dde727027b2759c',
       'php': '7.3',
       'behat_version': '10.3.2',
```

Comment:
Changing this to 10.4.0 will fix the tests.

Reply:
Ah, thanks! Will do.
silly me, I had the same in the community server.